test/loader_test/test_cl_runtime.c

```diff
@@ -4,7 +4,7 @@
 
 extern cl_command_queue command_queue;
 
-cl_int ret_val;
+static cl_int ret_val;
```

Comment:
Another one option is to add explicit initializers. Please let me know it that is preferred way.